cmd/license-detector/main.go

```diff
@@ -1,133 +1,97 @@
+// license-detector prints the most probable licenses for a repository
+// given either its path in the local file system or a URL pointing to
+// the repository.
 package main
 
 import (
+	"encoding/json"
+	"flag"
 	"fmt"
+	"net/url"
 	"os"
 	"sort"
-	"strings"
-
-	"encoding/json"
-	"github.com/spf13/pflag"
-	"gopkg.in/src-d/go-git.v4"
-	"gopkg.in/src-d/go-git.v4/plumbing/object"
-	"gopkg.in/src-d/go-git.v4/storage/memory"
-	"gopkg.in/src-d/go-license-detector.v1"
 	"sync"
+
+	"gopkg.in/src-d/go-license-detector.v1/detector"
+	"gopkg.in/src-d/go-license-detector.v1/detector/filer"
 )
 
 func main() {
-	var outputFormat string
-	pflag.StringVarP(&outputFormat, "format", "f", "text", "Output format, choose one of json, text.")
-	pflag.Parse()
-	if pflag.NArg() == 0 {
-		fmt.Fprintln(os.Stderr, "Usage: license-detector <path> [<path>...]")
-		os.Exit(1)
+	format := flag.String("f", "text", "Output format: json, text")
```

Comment:
It must be pflag. I am forcing posix compatibility, please revert.

Reply:
ok, fixed